pandas/tests/extension/list/array.py

```diff
@@ -1,138 +0,0 @@
-"""
```

Comment:
not a huge deal but id prefer if we got the tests passing with the implementation still living here, then moved the implementation in a separate PR

Reply:
Sure that's not a problem

Reply:
OK no big deal if its not easy